dpnp/tests/test_linalg.py

```diff
@@ -2144,6 +2147,218 @@ def test_check_finite_raises(self):
         assert_raises(ValueError, dpnp.linalg.lu_factor, a, check_finite=True)
 
 
+class TestLuSolve:
```

Comment:
Any plans to enable cupy tests?

Reply:
I have enabled cupyx tests in the last commit
Thank you!